cmake/inputs/Config.cmake.in

```diff
@@ -1,16 +1,17 @@
 #
 # This file attempts to locate the TileDB library. If found, the following
 # imported targets are created:
-#   - TileDB::tiledb_shared
-#   - TileDB::tiledb_static
+#   - TileDB::tiledb
```

Comment:
I don't know what this target is from this documentation. Same, different, what?

Reply:
Added documentation that `TileDB::tiledb` is present always.